app/src/main/java/com/hedvig/app/util/extensions/ContextExtensions.kt

```diff
@@ -127,34 +125,20 @@ fun Context.showAlert(
     @StringRes negativeLabel: Int = android.R.string.cancel,
     positiveAction: () -> Unit,
     negativeAction: (() -> Unit)? = null
-): AlertDialog =
-    AlertDialog
-        .Builder(this)
-        .setTitle(resources.getString(title))
-        .setPositiveButton(resources.getString(positiveLabel)) { _, _ ->
-            positiveAction()
-        }
-        .setNegativeButton(resources.getString(negativeLabel)) { _, _ ->
-            negativeAction?.let { it() }
-        }
+): androidx.appcompat.app.AlertDialog? =
+    MaterialAlertDialogBuilder(this)
         .apply {
+            setTitle(resources.getString(title))
+            setPositiveButton(resources.getString(positiveLabel)) { _, _ ->
+                positiveAction()
+            }
+            setNegativeButton(resources.getString(negativeLabel)) { _, _ ->
+                negativeAction?.let { it() }
+            }
             message?.let { setMessage(it) }
         }
         .show()
         .apply {
-            getButton(AlertDialog.BUTTON_POSITIVE).apply {
-                background = ColorDrawable(Color.TRANSPARENT)
-                setTextColor(resources.getColor(R.color.purple))
-            }
-            getButton(AlertDialog.BUTTON_NEGATIVE).apply {
-                background = ColorDrawable(Color.TRANSPARENT)
-                setTextColor(resources.getColor(R.color.purple))
-            }
-            getButton(AlertDialog.BUTTON_NEUTRAL).apply {
-                background = ColorDrawable(Color.TRANSPARENT)
-                setTextColor(resources.getColor(R.color.purple))
-            }
-            window?.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
             getButton(AlertDialog.BUTTON_POSITIVE)?.isAllCaps = false
```

Comment:
While we're at it, let's go all the way with theming and use the theme to control this as well.